### DISCUSSION TITLE
Basic Username/Password for Schema Registry

### DISCUSSION THREAD

**Discussion by RPalejiya** [LGouellec/streamiz#93] [Q&A] [answered]
I am trying to make stream work with Schema registry which requires basic authentication to connect. I wonder how to provide that config. I am trying with Add but no luck. 
```

var config = new StreamConfig<StringSerDes, StringSerDes>();
config.SchemaRegistryUrl = "https://dev01-app01-schema-registry.aura.us";
config.AutoRegisterSchemas = true;
**config.Add("schema.registry.basic.auth.credentials.source", "USER_INFO"); 
config.Add("schema.registry.basic.auth.user.info", "producer:secret");**
```

I end up with error regardless if I add user info or not
System.Net.Http.HttpRequestException: [https://dev01-app01-schema-registry.aura.us/] HttpRequestException: Unauthorized

**Comment by LGouellec** [comment, marked as answer]
Hi @RPalejiya, 

Yes it's normal. This settings will be available in next release (1.2.0, planned January 2022).
It's also available on develop branch on my repository : https://github.com/LGouellec/kafka-streams-dotnet/blob/669eaaad0169b8726582af98dc5edf9da384797b/core/SchemaRegistryConfig.cs#L66
You can clone my repo just for test.

Regards,

**Reply by RPalejiya** [reply]
Thank you very much for the Prompt reply and much appreciated your effort as whole. I will give a try and cant wait for new release. 

Cheers,
Ravi

**Reply by LGouellec** [reply]
Hi,

1.2.0 is released since some days :) 

Cheers
